src/tools/xtp_run.cc

```diff
@@ -107,18 +103,18 @@ bool XtpRun::EvaluateOptions() {
   for (const auto& calc : xtp::Calculators().getObjects()) {
 
     if (calc_string[0].compare(calc.first) == 0) {
-      cout << " This is a XTP app" << endl;
+      std::cout << " This is a XTP app\n";
```

Comment:
I think this can go. It is a remnant of when we were wrapping CTP calculators too